src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -34,14 +36,14 @@ public void equals() {
         NameContainsKeywordsPredicate secondPredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("second"));
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        FindCommand findFirstCommand = new FindCommand(firstPredicate, null, null);
+        FindCommand findSecondCommand = new FindCommand(secondPredicate, null, null);
```

Comment:
`null` creates confusion and affects readability as it can be anything. since `null` is essentially unused argument, 
suggest creating overloaded constructors to handle it.
```
public FindCommand(NameContainsKeywordsPredicate a, PhoneContainsKeywordsPredicate b) {
    this(a, b, null);
}

// all other possible permutations
... 
```